server/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteResponse.java

```diff
@@ -44,7 +44,7 @@
         explanations = RoutingExplanations.readFrom(in);
     }
 
-    public ClusterRerouteResponse(boolean acknowledged, ClusterState state, RoutingExplanations explanations) {
+    ClusterRerouteResponse(boolean acknowledged, ClusterState state, RoutingExplanations explanations) {
```

Comment:
Just a revert of making this public yesterday now that it's not needed for tests any longer.